reporting/api/src/main/java/org/jboss/windup/reporting/model/ClassificationModel.java

```diff
@@ -97,4 +104,12 @@
     @SetInProperties(propertyPrefix = "tag")
     Set<String> getTags();
 
+    abstract class Impl implements ClassificationModel, JavaHandlerContext<Vertex>
```

Comment:
Right... I'm saying that it would be better to copy the method declaration to the "ClassificationModel" and attach the annotation there. Effectively @JavaHandler is specifying an implementation, so semantically it should exist on the implementing class.